### PR TITLE
Cosmetic changes

### PR DESCRIPTION
**Pull request by dqwiki**

### CODE REVIEW

File: resources/views/appeals/appeal.blade.php [hunk 1]
```diff
@@ -113,6 +113,17 @@ class="btn btn-warning">
                         <div class="col-md-7">
                             @if($info->privacyreview!=0 && $info->status=="PRIVACY" && $perms['admin'])
                                 <div class="row">
+                                    @if ($info->privacyreview==1 || $info->privacyreview==2)
+                                        <div class="alert alert-primary" role="alert">
+                                            @if ($info->privacyreview==1)
+                                                It has been requested that this appeal be hidden from public view and only visibleto
```
Comment (supertassu) [inline resources/views/appeals/appeal.blade.php]:
```suggestion
                                                It has been requested that this appeal be hidden from public view and only visible to
```

File: resources/views/appeals/appeal.blade.php [hunk 2]
```diff
@@ -113,6 +113,17 @@ class="btn btn-warning">
...
+                                                administrators.
+                                            @elseif ($info->privacyreview==2)
+                                                It has been requested that this appeal be oversighted and only availible to those on
```
Comment (supertassu) [inline resources/views/appeals/appeal.blade.php]:
```suggestion
                                                It has been requested that this appeal be oversighted and only available to those on
```

**Review by supertassu — COMMENTED** [review]
Couple of typos

**Review by supertassu — APPROVED** [review]

**Comment by dqwiki** [comment]
I'm going to mark this for rereview and open a new one for a new week of changes after this is merged.

**Review by supertassu — APPROVED** [review]

**Pull request merged**